metadata-ingestion/tests/integration/powerbi/test_powerbi.py

```diff
@@ -819,6 +819,8 @@ def test_powerbi_ingest_urn_lower_case(
                 "type": "powerbi",
                 "config": {
                     **default_source_config(),
+                    "env": "PROD",
+                    "platform_instance": "PROD",
```

Comment:
platform_instance = "PROD" will be confusing. can we do something like "myPlatformInstance"